extensions/aerial-photo/schema.json

```diff
@@ -42,13 +42,16 @@
       "type": "object",
       "allOf": [
         {
-          "required": ["type"],
+          "required": ["type", "summaries"],
```

Comment:
Should we specify the summary types in the extensions (e.g. aerial-imagery) as done below but **require** the summaries and specific fields in the **historical imagery** schema rather than in these ones? In case others don't want to use summaries (even though they are recommended). @MDavidson17 @dwsilk

Reply:
Chatted with @dwsilk good to go as it is, we can make this change later.